automobile/road-segments.md

```diff
@@ -252,6 +270,41 @@ only the specific ones will be explained.
 - `heigthStep`: Defines the step for the road to complete one lap.
 - `subdivision`: Defines the subdivision of the helicoid.
 
+### Crossroad
+
+The `Crossroad` PROTO represents a Crossroad.
+
+%figure "Crossroad intersection"
+
+![crossroad.png](images/crossroad.png)
+
+%end
+
+```
+Crossroad {
+  SFVec3f    translation             0 0 0
+  SFRotation rotation                0 1 0 0
+  SFString   id                      ""
+  MFVec3f    shape                   [ 0 0 0, 1 0 0, 0 0 1]
+  MFString   connectedRoadIDs        []
+  SFBool     crossroadBoundingObject FALSE
```

Comment:
You're right, I have done this to match the `Road.roadBoundingObject` field, but this has been done because this field is split there. Other PROTOs like RoadIntersection have simply a boolean `RoadIntersection.boundingObject`. I will go on with the same, in order to be consistent.